src/react/stylesheets/_variables.scss

```diff
@@ -9,6 +9,9 @@ $large-spacing: 15px;
 
 // Typography:
 $base-font-color: #333;
+$color-text-alt: $cc-color-dim-gray;
+$color-text-link: #2c84bc;
```

Comment:
@developish - I really recommend that we establish the new color palette that the design team desires under the new React CSS. If you look at the old [colors.scss](https://github.com/sis-berkeley-edu/calcentral/blob/master/src/assets/stylesheets/colors.scss) you'll see we used [HTMLCSSColor.com](https://www.htmlcsscolor.com/hex/2C84BC) to help us give the colors neutral names not specific to where/how they're used.

Reply:
I always wondered where those names came from....

Reply:
@developish - If we want to go with an open source naming convention, maybe this would be better - [Find a name for almost any hex color](https://www.abeautifulsite.net/find-a-name-for-almost-any-hex-color)

- [Name That Color](http://chir.ag/projects/ntc/)
- [Name That Color - Mariner](http://chir.ag/projects/name-that-color/#2C84BC)

Reply:
For now, I think that we should continue to use HTMLCSSColor.com - especially considering that doing so will expose multiple colors that are _very_ close to each other. It doesn't have a unique name for every single possible color, there's a bit of a snap-to-grid. This brings up an issue that we should further discuss in our design meeting.

I'm open to doing a rename to the open source version in another PR later.